catalog/nsili/catalog-nsili-source/src/main/java/com/connexta/alliance/nsili/source/NsiliSource.java

```diff
@@ -885,6 +887,16 @@ public void setExcludeSortOrder(Boolean excludeSortOrder) {
         }
     }
 
+    public Boolean getSwapCoordinates() {
+        return swapCoordinates;
+    }
+
+    public void setSwapCoordinates(Boolean swapCoordinates) {
+        if (swapCoordinates != null) {
```

Comment:
Since you default the metatype value, if use the primitive, this will never be null.